src/index.js

```diff
@@ -226,22 +225,25 @@ Hooks.on("createToken", (newNpc) => {
         // Add hit dice to formula
         hitDiceFormula += hitDice[prop].total + prop;
         // Increment Hit Dice Count total
-        hitDiceTotalCount++
+        hitDiceTotalCount++;
       }
```

Comment:
Probably a good idea to create a getter for this in the `Actor5e` class instead of us constructing the formula in potentially several places.

Reply:
```js
Object.entries(hd).reduce((acc, [dieType, hitDie]) =>{
   if (hitDie.total) return acc.concat(`${hitDie.total}${dieType}`);
    return acc
}, []).join("+")
```

Would be a better way to construct it.

Reply:
Would prefer this as per comments on Discord:
```js
Object.entries(hd).reduce((acc, [dieType, hitDie]) =>{
  if (hitDie.total) acc.push(`${hitDie.total}${dieType}`);
  return acc;
}, []).join("+");
```